packages/angular/build/src/builders/dev-server/vite-server.ts

```diff
@@ -470,7 +475,7 @@ async function handleUpdate(
           // are not typically reused across components.
           const componentIds = usedComponentStyles.get(filePath);
           if (componentIds) {
-            return componentIds.map((id) => ({
+            return [...componentIds].map((id) => ({
```

Comment:
Consider: Prefer `Array.from` to be more clear about your intent.

https://techhub.social/@develwithoutacause/111819710786873448